src/lib.rs

```diff
@@ -2103,12 +2111,23 @@ impl Build {
             None => false,
         };
 
-        let is_sim = match target.split('-').nth(3) {
+        let is_arm_sim = match target.split('-').nth(3) {
```

Comment:
This should still be named `is_sim` -- As it is, further down we have x86-specific logic in an `if is_arm_sim` branch.